content/en/llm_observability/sdk.md

```diff
@@ -104,6 +104,12 @@ LLMObs.enable(
 : optional - _string_
 <br />The name of the service used for your application. If not provided, this defaults to the value of `DD_SERVICE`.
 
+### AWS Lambda setup
+
+Follow the setup instructions for the [Datadog-Python and Datadog-Extension][14] layers and LLM Observability will be automatically enabled.
+
+In order for these serverless layers to properly instrument and flush traces from your serverless application, specify `DD_LLMOBS_ENABLED` in your environment. This applies to both manual and auto-instrumented use cases.
```

Comment:
Are there other configurations that are required to run LLMObs besides the enabled flag?